archeryutils/handicaps/handicap_functions.py

```diff
@@ -158,7 +115,37 @@ def print_handicap_table(
         )
         print("Done.")
 
-    return None
```

Comment:
Isn't it good practice to have a return None at the end of the function (print_handicap_table)?

Reply:
I think it might just be personal preference. I usually exclude it, since you don't tend to return from `void` functions in C/C++, and I picked up most of my bad habits there. I don't think it changes behaviour either way. I guess that it might help the reader to see when one function ends and another begins?

Reply:
Yes, I prefer it, so if you can leave it in that would be great, thanks.

Reply:
I added it back in before I read your comment, but as a plain `return` instead of `return None`. I'd argue there's a difference in intent between the two: plain `return` says "the function exits here", while `return None` says "I'm explicitly returning a None which should be handled elsewhere in the code". Up to you though.